### ISSUE TITLE
Cannot install Net 7.0 since reached EOL.

### ISSUE THREAD

**Issue by jsalaz-Gramps** [JamesCJ60/Universal-x86-Tuning-Utility#262]
**Describe the bug**

The installation of the installer, asks for .net 7.0, but this entry is out of date, and is no longer supported.

**To Reproduce**
Steps to reproduce the behaviour:
1. Attempt to install
2. When program asks to download .Net 7, it fails.
3. Program exits.

**Expected behavior**

Allow computer to install the .Net 9 platform, and continue installation.

**Screenshots**

**Desktop (please complete the following information):**
 
OS: Windows 11 24H2

**Comment by dartraiden** [comment]
Also, it's weird that the installer first requires .NET 7.0, and then the application requires 8.0

**Comment by IAmEchino** [comment]
Yeah something's definitely bugged, I installed 8.0 (the version that the changelog says is compatible) and it still is asking for 7.0.

**Comment by jsalaz-Gramps** [comment]
Let's just hope it is upgraded, because of the fast pushout of .Net 7.0.
It seems like it was rushed.  >nert 8.0 is better anyway.

**Comment by LeGraon** [comment]
Hello. I have the same problem with UXTU V2.4.2 on Windows 11 24H2. The installer is still asking for .NET 7.

**Comment by jsalaz-Gramps** [comment]
I figure that the project is dead, since there has been no response to our
concerns for 6-8 months.

**Comment by LeGraon** [comment]
The project is not dead since there was an update to the version 2.4.5 one hour ago. New version but same issue...

**Comment by jsalaz-Gramps** [comment]
OK!

**Comment by LeGraon** [comment]
Actually I first installed .Net 7 then .Net 8 and it works well.

**Comment by JamesCJ60** [comment]
Hi, this issue has been fixed as of the latest beta builds.
